composer.json

```diff
@@ -24,7 +24,7 @@
         "ocramius/package-versions": "^1.4",
         "phar-io/version": "^3.0.3",
         "phpstan/phpdoc-parser": "^0.4.9",
-        "phpstan/phpstan": "^0.12.64",
+        "phpstan/phpstan": "^0.12.64, <0.12.70",
```

Comment:
There is some bug with abstract class resolution in PHPStan 0.12.70, so we need to fallback.
It can be spotted in `Symplify\PHPStanRules\Rules\NoParentMethodCallOnEmptyStatementInParentMethodRule` while parsing parent method content by php-parser